app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,16 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
- [ ] Your app is not running due to a configuration error. Kindly ensure to troubleshoot this error so that cloned versions of your app run correctly.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/143661253-3573ff01-10f0-455b-a701-c2710dc69067.png)

![image](https://user-images.githubusercontent.com/12745474/143661573-250bd4e1-9187-410c-9892-e8ce97332c86.png)

Reply:
I'm sorry, I changed the default pipeline that `rails server` uses to run the project and compile assets on live. I did this to be able to include `tailwindcss` as specified [in this repository.](https://github.com/rails/cssbundling-rails)

To the next reviewer, to avoid this configuration error run the following commands after cloning the repo.

- Run `bundle install`
- Run `rails assets:precompile`
- Run `rails db:setup`
- Run `rails server` or `rails s`

The README file has been updated to reflect these specifications.